src/components/Encounter/EncounterCommandDialog.tsx

```
@@ -0,0 +1,486 @@
+                    <span className="flex-1">{action.label}</span>
+
+                    {action.shortcut && (
+                      <CommandShortcut className="ml-2 text-xs text-gray-500 bg-white border border-gray-200 shadow-xs px-1.5 py-0.5 rounded">
+                        {action.shortcut}
+                      </CommandShortcut>
+                    )}
+                  </CommandItem>
+                );
+              })}
+            </CommandGroup>
+            <CommandSeparator />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)